packages/provider/src/HocuspocusProviderWebsocket.ts

```diff
@@ -369,7 +367,8 @@ export class HocuspocusProviderWebsocket extends EventEmitter {
     }
 
     // No message received in a long time, not even your own
-    // Awareness updates, which are updated every 15 seconds.
+    // Awareness updates, which are updated every 15 seconds
+    // if awareness is enabled.
```

Comment:
will the connection actually survive this (checkConnection logic) if there is no awareness message? 🤔 If no update is happening on server/client, and no awareness is received, I think this method will assume the connection is dead?

Reply:
The connection will auto-close if there are no messages from any provider for the timeout duration. However, the idea is that you only need a single provider to enable Awareness, not hundreds when there are hundreds of documents.

Reply:
makes sense, thx. I'll add an explanation to the parameter.